server/api/ballot_manifest.py

```diff
@@ -125,11 +125,7 @@ def process() -> None:
         )
 
         columns = [
-            CSVColumnType(
-                CONTAINER,
-                CSVValueType.TEXT,
-                required_column=is_counting_group_required,
-            ),
+            CSVColumnType(CONTAINER, CSVValueType.TEXT, required_column=False),
```

Comment:
It might be slightly confusing to readers that we indicate the column isn't required, but later on enforce it in L158. Thoughts on allowing an error message override to be specified in the `CSVColumnType`? I could see it being overkill, so if you agree, maybe just add a comment here on why we mark `required_column=False` and enforce with special logic later.

Reply:
Good point, I'll add a comment!